conduct.md

```diff
@@ -116,15 +119,23 @@ how long a decision will take to implement.
     * Report errors or bugs to the appropriate team members.
     * When a choice is disputed, use Github issue tracker and let team members
       share their opinion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the decision making as a team.
+    * Defer to the Professor or Technical Leaders when unsure about tasks or implementation.
+    * Use efficient communication within and outside of teams.
+    * Communicate with the appropriate team member about any changes that is needed to be done to any project.
+    * Use efficient communication within and outside of teams
+    * Stay respectful and openminded when discussing decisions and ideas.
+    * Communicate with the appropriate team member about any changes that is needed to be done to any project.
```

Comment:
Efficient communication and communicating with the appropriate team member were mentioned twice in this list. It would also be great if you can explain what exactly do you mean by efficient communication